Templates/administration/administration.html.twig

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
```

Comment:
I think you should create a base_admin.html.twig if it's different from base.html.twig to avoid duplications ^^,